### DISCUSSION TITLE
[DOCKER] `latest` tag ambiguity

### DISCUSSION THREAD

**Discussion by mgxd** [nipreps/fmriprep#2640] [Q&A]
Seeing as how we plan on supporting two series at once (LTS, 21/22), we may need to rethink the `latest` tag, since it currently just points to the last release (for example, `latest` could either be 20.2.7 or 21.0.0). 

Two options come to mind:
- use `latest` for the latest series only (remove `latest` docker push to from `maint/` branches)
- use the series prefix, i.e. `latest-20.2.x`

**Comment by effigies** [comment]
I would remove it from the older `maint/` branches once a new series is released. We discourage people from using it; updating it is mostly to keep people from running `docker pull nipreps/fmriprep` and getting an extremely old version.
